tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-    py{27,34,35,36}-django111
-    py{34,35,36}-django20
+    py{27,34,35,36,37}-django111
```

Comment:
Django 1.11.x does not officially support Python 3.7.x.

Reply:
oh yeah! You are right. Thx :+1: